<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}

</string>
       </property>
       <property name="text">
        <string>Select port:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portsComboBox">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox
{
border: 1px solid gray;
border-radius: 3px;
padding: 1px 18px 1px 3px;
background-color:rgb(255, 178, 42);
color: white;
	font: 30px &quot;MS Shell Dlg 2&quot;;
}
QComboBox QAbstractItemView{background: rgb(255, 178, 42);}

QComboBox::item:selected
{
    background-color: rgb(255, 190, 50);
    color: rgb(0, 0, 0);
}
QComboBox::drop-down {
    width: 70px;
}

QComboBox::indicator{
    background-color:transparent;
    selection-background-color:transparent;
    color:transparent;
    selection-color:transparent;
}</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rescanPortsButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color: white;
background-color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover:!pressed
{
background-color: rgb(255, 190, 50);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/res/img/refresh.png</normaloff>:/img/res/img/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="connectedJudgesLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}

</string>
       </property>
       <property name="text">
        <string>Connected judges:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startStopButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color: white;
background-color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover:!pressed
{
background-color: rgb(255, 190, 50);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/res/img/play.png</normaloff>:/img/res/img/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newFightButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color: white;
background-color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover:!pressed
{
background-color: rgb(255, 190, 50);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/res/img/clean.png</normaloff>:/img/res/img/clean.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="competitor1points">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>299</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>3000000</width>
         <height>3000000</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
	font: 75 400pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: 'AlignCenter | AlignCenter';
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="competitor2points">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300000</width>
         <height>300000</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 255);
	font: 75 400pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: 'AlignCenter | AlignCenter';
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
